db_populate.py

```diff
@@ -0,0 +1,224 @@
+# This file is used to populate the database(mainly for testing)
+from lemur.utility import *
```

Comment:
I'd avoid importing * and instead use explicit imports. Explicit is better than implicit.